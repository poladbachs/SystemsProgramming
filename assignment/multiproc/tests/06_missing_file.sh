"$1" ./tests/lorem_ipsum.txt not-exising-file ./tests/directory >/dev/null
echo "Exit code is: $?"
