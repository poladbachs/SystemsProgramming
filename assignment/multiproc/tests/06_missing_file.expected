Failed to process: not-exising-file
Exit code is: 1
